assets/js/base/components/cart-checkout/button/style.scss

```diff
@@ -5,7 +5,7 @@
 	display: flex;
 	font-size: inherit;
 	font-weight: bold;
-	min-height: rem(48px); // 16px * 3 = 48px, same height as text-input
+	min-height: 3em;
```

Comment:
Why not using the new `em` function (`em(48px)`) (assuming there's good reason to switch from `rem` to `em`)?

Reply:
In pb0Spc-Cv-p2's comments, @mikejolley raised the concern that it might be confusing for developers to write px units but see they are `em`'s in the resulting CSS. In this concrete case, if you are debugging a theme with a font size of 14px and want to know how high that element will be, I think it's easier to rationale "`3em = 14*3 = 42`" than thinking "`48 / 16 = 3, 14 * 3 = 42`".

I still think the `em()` mixin makes sense to convert px variables to `em` values. But in other cases it might be more intuitive to use `em` than `px`. Does that make sense?